src/homework11/Printable.java

```diff
@@ -0,0 +1,12 @@
+package homework11;
+
+import java.util.function.Function;
+
+public interface Printable {
+     static void main(String[] args) {
+         Function<String, String> print = String::toUpperCase;
```

Comment:
It is a good one but you should write lambda for your own FunctionalInterface. 
So PLs create Printable from HW8 as FunctionalInterface and rewrite such a code)